homeassistant/components/simplefin/strings.json

```diff
@@ -6,6 +6,13 @@
         "data": {
           "api_token": "Claim Token or Access URL"
         }
+      },
+      "reauth_confirm": {
+        "title": "SimpleFin - Re-authentication Required",
+        "description": "Please enter either a new Claim Token or an Access URL.",
+        "data": {
+          "api_token": "Claim Token or Access URL"
```

Comment:
```suggestion
          "api_token": "[%key:components::simplefin::config::step::user::data::api_token%]"
```

Reply:
I'm going to say no on this one -> The documentation specifically uses the terms `Claim Token` and `Access URL`.

A `claim token` can be exacted (ONLY ONCE) for an `access url` and an `access url` can be used many times.